lib/pbench/server/indexing_tarballs.py

```diff
@@ -513,6 +516,8 @@ def sighup_handler(*args):
                                 Audit.create(
                                     root=audit, status=doneness, attributes=attributes
                                 )
+                            if lock:
+                                lock.release()
```

Comment:
How could `lock` be false-y at this point?

Reply:
The nested `try` blocks in this module are messy, and the code here tries to be really careful about assuming the arrival state.